video/video_library_interface_review.rb

```diff
@@ -0,0 +1,184 @@
+# DO NOT MERGE - this file for comments only
+
+def analyze_labels_gcs path:
+  # [START analyze_labels_gcs]
+  # path = "Path to a video file on Google Cloud Storage: gs://bucket/video.mp4"
+
+  require "google/cloud/video_intelligence/v1beta1"
+
+  video_client = Google::Cloud::VideoIntelligence::V1beta1::VideoIntelligenceServiceClient.new
+  features     = [:LABEL_DETECTION]
+
+  # Register a callback during the method call
+  operation = video_client.annotate_video path, features do |operation|
+    raise operation.results.message? if operation.error?
+    puts "Finished Processing."
+
+    # first result is retrieved because a single video was processed
+    annotation_result = operation.results.annotation_results.first
```

Comment:
Hmm. What attributes does a Result object have besides `annotation_results`??

This is weird. I'd figure the `operation.results` would already be an array of `AnnotationResult` or some such

Reply:
That's the only field; it's defined in the proto here: https://github.com/googleapis/googleapis/blob/master/google/cloud/videointelligence/v1beta2/video_intelligence.proto#L310

Reply:
Chiming in:  You can define wild card gcs:// paths for multiple videos as noted in [AnnotateVideoRequest](https://cloud.google.com/video-intelligence/docs/reference/rpc/google.cloud.videointelligence.v1#annotatevideorequest)